README.txt

```diff
@@ -0,0 +1,40 @@
+macrosheds sites are organized by network (e.g. LTER), domain (e.g. HBEF), and site (e.g. W1)
```

Comment:
Just a flag, but this will be perfect for spencer to start converting to bookdown. I assume that is your goal!

Reply:
cha!